snippet-extractor-output/snippets.yaml

```diff
@@ -13089,6 +13105,450 @@
+        await context.sync();
+        const slide2 = context.presentation.slides.getItemAt(1);
+        const slide4 = context.presentation.slides.getItemAt(3);
+        const slide5 = context.presentation.slides.getItemAt(4);
+        slide2.load("id");
+        slide4.load("id");
+        slide5.load("id");
+        await context.sync();
+        context.presentation.setSelectedSlides([slide2.id, slide4.id, slide5.id]);
+        await context.sync();
+      });
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/powerpoint/slide-management/get-set-slides.yaml
+
+    // Sets selection to the slides that were saved.
+      await PowerPoint.run(async (context) => {
+        context.presentation.setSelectedSlides(savedSlideSelection);
+        await context.sync();
+      });
+'PowerPoint.Slide#getItemAt:member(1)':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/powerpoint/slide-management/get-set-slides.yaml
+
+    // Selects slides 2, 4, and 5.
+      await PowerPoint.run(async (context) => {
+        context.presentation.load("slides");
+        await context.sync();
+        const slide2 = context.presentation.slides.getItemAt(1);
+        const slide4 = context.presentation.slides.getItemAt(3);
+        const slide5 = context.presentation.slides.getItemAt(4);
+        slide2.load("id");
+        slide4.load("id");
+        slide5.load("id");
+        await context.sync();
+        context.presentation.setSelectedSlides([slide2.id, slide4.id, slide5.id]);
+        await context.sync();
+      });
+'PowerPoint.Slide#setSelectedShapes:member(1)':
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/powerpoint/shapes/get-set-shapes.yaml
+
+    // Reselects shapes that were saved previously.
+
+    await PowerPoint.run(async (context) => {
+      const slide1 = context.presentation.slides.getItem(savedSlideSelection[0]);
+      await context.sync();
+      slide1.setSelectedShapes(savedShapeSelection);
+      await context.sync();
+    });
+  - >-
+    // Link to full sample:
+    https://raw.githubusercontent.com/OfficeDev/office-js-snippets/prod/samples/powerpoint/shapes/get-set-shapes.yaml
+
+    // Selects the first two shapes on the selected slide.
+
+    await PowerPoint.run(async (context) => {
+      context.presentation.load("slides");
+      await context.sync();
+      const slide1 = context.presentation.slides.getItemAt(0);
+      slide1.load("shapes");
+      await context.sync();
+      const shapes = slide1.shapes;
+      const shape1 = shapes.getItemAt(0);
+      const shape2 = shapes.getItemAt(1);
+      shape1.load("id");
+      shape2.load("id");
+      await context.sync();
+      slide1.setSelectedShapes([shape1.id, shape2.id]);
+      await context.sync();
+    });
+'PowerPoint.Shape#fill:member(1)':
```

Comment:
`fill` is a property so in the spreadsheet, its "Member ID" column should be blank

Reply:
should be fixed now.